package-lock.json

```diff
@@ -46932,7 +46943,7 @@
         "@googleforcreators/patterns": "*",
         "@googleforcreators/react": "*",
         "draft-js": "^0.11.7",
-        "draft-js-export-html": "git+ssh://git@github.com/swissspidy/draft-js-export-html.git#366642f1c6e5090f24f7a66a357328115e4a6a62",
+        "draft-js-export-html": "https://github.com/swissspidy/draft-js-export-html",
```

Comment:
@swissspidy, I have to update this file because I've added a package, but these changes seem unwanted?

Reply:
Should be fine, npm sometimes likes to switch depending on git config and stuff.